shared/processing/schemaGetters.ts

```diff
@@ -0,0 +1,354 @@
+import {
+    ContentItem,
+    Fields,
+} from 'kentico-cloud-delivery';
+import { ZapiDiscriminator } from '../models/zapi_discriminator';
+import { ZapiDiscriminatorMapItem } from '../models/zapi_discriminator__map_item';
+import { ZapiPropertyReferencingASchema } from '../models/zapi_property_referencing_a_schema';
+import { ZapiSchemaAllof } from '../models/zapi_schema__allof';
+import { ZapiSchemaAnyof } from '../models/zapi_schema__anyof';
+import { ZapiSchemaArray } from '../models/zapi_schema__array';
+import { ZapiSchemaBoolean } from '../models/zapi_schema__boolean';
+import { ZapiSchemaInteger } from '../models/zapi_schema__integer';
+import { ZapiSchemaNumber } from '../models/zapi_schema__number';
+import { ZapiSchemaObject } from '../models/zapi_schema__object';
+import { ZapiSchemaOneof } from '../models/zapi_schema__oneof';
+import { ZapiSchemaString } from '../models/zapi_schema__string';
+import {
+    IDiscriminator,
+    IDiscriminatorMapItem,
+    IPreprocessedData,
+    IPropertyReferencingASchema,
+    ISchemaAllOf,
+    ISchemaAnyOf,
+    ISchemaArray,
+    ISchemaBoolean,
+    ISchemaElements,
+    ISchemaInteger,
+    ISchemaNumber,
+    ISchemaObject,
+    ISchemaObjectPropertyElements,
+    ISchemaOneOf,
+    ISchemas,
+    ISchemaString,
+    IWrappedItem,
+    ZapiSchemas,
+} from '../types/dataModels';
+import {
+    getFromLinkedItems,
+    processLinkedItemsElement,
+    processMultipleChoiceElement,
+    processTaxonomyElement,
+} from '../utils/processElements';
+import {
+    getSystemProperties,
+    getWrappedData,
+    processItemsInto,
+} from './common';
+import RichTextField = Fields.RichTextField;
+
+export const getSchemaDataFromLinkedItemElement = (
+    items: ZapiSchemas[],
+    dataBlob: IPreprocessedData,
+    linkedItems: ContentItem[],
+): Array<IWrappedItem<ISchemas>> => {
+    const schemas = [];
+
+    items.map((schema: ZapiSchemas) => {
+        schemas.push(getSchemaData(schema, dataBlob, linkedItems));
+    });
+
+    return schemas;
+};
+
+export const getSchemaDataFromRichTextElement = (
+    field: RichTextField,
+    dataBlob: IPreprocessedData,
+    linkedItems: ContentItem[],
+): Array<IWrappedItem<ISchemas>> => {
+    const schemas = [];
+
+    field.linkedItemCodenames.map((codename: string) => {
+        const schema = getFromLinkedItems<ZapiSchemas>(codename, linkedItems);
+
+        schemas.push(getSchemaData(schema, dataBlob, linkedItems));
+    });
+
+    return schemas;
```

Comment:
I know you had no chance to fix this (since my last review comment in this first review), but seriously. This is literaly 6 lines longer method than it could be while maintaining same level of readability...